src/Translators/Rules.php

```diff
@@ -80,6 +80,18 @@ public static function fromColumn(string $context, Column $column)
             array_push($rules, 'unique:' . $context . ',' . $column->name());
         }
 
+        if (Str::contains($column->dataType(), 'decimal')) {
```

Comment:
These could all be refactored to `in_array` as the `dataType` will always been one of those.